libbeat/common/cfgwarn/removed.go

```diff
@@ -0,0 +1,48 @@
+package cfgwarn
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/joeshaw/multierror"
+)
+
+func CheckRemoved5xSettings(cfg *common.Config, settings ...string) error {
```

Comment:
Works for me for the moment. i think we can use such a method in a more generic way as this is no the only time we remove settings. Something like `CheckRemoveSettings()` and the one here with 5x is just a proxy for it.